nextflow_schema.json

```diff
@@ -109,6 +130,15 @@
                     "help_text": "If you would like to limit your analysis to specific regions of the genome, you can pass those regions in a bed file using this option",
                     "hidden": true
                 },
+                "sentieonbwa_index": {
```

Comment:
I am a bit confused here, is it the ref index that SENTIEON_BWAINDEX creates?

Reply:
This is the parameter that stores the path to the bwa index files to be used with sentieon - referred to as params.sentieonbwa_index in the workflow. SENTIEON_BWAINDEX will only run if the value of params.sentieon_bwa_index is empty. https://github.com/nf-core/raredisease/blob/a81900561c3fa38fa61d66cfee681ecf7212cd80/conf/modules.config#L72-L80